packages/create-turbo/package.json

```diff
@@ -24,7 +24,7 @@
   },
   "dependencies": {
     "chalk": "4.1.2",
-    "commander": "^10.0.0",
+    "commander": "^11.0.0",
```

Comment:
Updating to use same version across all our CLIs. Nothing breaking is relevant for us:

https://github.com/tj/commander.js/releases/tag/v11.1.0

Reply:
Wish pnpm had workspace packages like in cargo...